docs/metadata/4.0.0.jsx

```diff
@@ -197,8 +197,8 @@ insert(docs, "vertx-mongo-client",
   {
     id: "vertx-sql-client-templates",
     name: "SQL Client Templates",
-    description: `A small library designed to facilitate the execution and data
-      manipulation of SQL queries.`,
+    description: `一个轻量级的库，旨在增强SQL查询
```

Comment:
这样换行会在 “查询” 和 “和数据” 之间多一个空格的。  
改成在逗号那里换行吧：
```suggestion
    description: `一个轻量级的库，
```